hera_pspec/pstokes.py

```diff
@@ -137,23 +226,43 @@ def _combine_pol(uvd1, uvd2, pol1, pol2, pstokes='pI', x_orientation=None):
     assert pol2 in pol_weights[pstokes], \
         "pol2 {} not used in constructing pstokes {}".format(pol2_str, pstokes_str)
 
+    # assert pol_convention makes sense
+    if pol_convention not in ['avg', 'sum']:
+        raise ValueError('pol_convention must be avg or sum.')
+
+    # checks on input lists
+    for a_list in [data_list, flags_list, nsamples_list]:
+        if a_list is None:
+            continue
+        assert len(a_list) == 2, \
+            "Cannot combine more than two arrays."
```

Comment:
Or fewer

Reply:
Ok, I added the option to only feed a list of one element (e.g. `data_list=[uvd1.data_array]`), in which case it is duplicated and combined to itself to form the pseudo-Stokes pol